cli/run.py

```diff
@@ -58,9 +71,14 @@ def kill_process_group_afterwards():
 @click.argument(
     "script_path", type=click.Path(exists=True), metavar="<script>", required=True
 )
-@click.argument("script_args", nargs=-1, type=click.UNPROCESSED)
-def run_experiment(envision, envision_port, script_path, script_args):
+@click.argument("scenario_path", nargs=-1, type=click.UNPROCESSED)
```

Comment:
I agree with @Adaickalavan's take on this. Please make sure that the change does not prevent forwarding options to the underlying script.